### PR TITLE
ui: Honour default.ui.page.size

### PR DESCRIPTION
**Pull request by davidjumani**
Fixes https://github.com/apache/cloudstack/issues/5338

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan ui

File: engine/schema/src/main/resources/META-INF/db/schema-41500to41510.sql [hunk 1]
```diff
@@ -132,3 +132,5 @@ INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_vers
 
 -- Add support for windows2019srvNext_64Guest from VMware 7.0.1.0
 INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(),'VMware', '7.0.1.0', 'windows2019srvNext_64Guest', 276, now(), 0);
+
+UPDATE `cloud`.`configuration` SET `value` = 20, `default_value` = 20 WHERE `name` = 'default.ui.page.size';
```
Comment (shwstppr) [inline engine/schema/src/main/resources/META-INF/db/schema-41500to41510.sql:136]: @davidjumani won't we have to create a 41510to41520 upgrade for it to work for 4.15.1 to 4.15.2 upgrade?

**Review by utchoang — APPROVED** [review]
UI LGTM!

File: engine/schema/src/main/resources/META-INF/db/schema-41500to41510.sql (hunk 1 above)
Comment (rhtyd) [reply engine/schema/src/main/resources/META-INF/db/schema-41500to41510.sql:136]: If config key exists we shouldn't need it

Comment (davidjumani) [reply engine/schema/src/main/resources/META-INF/db/schema-41500to41510.sql:136]: Yes, will make the change

**Comment by davidjumani** [comment]
Converting to draft until the sql code is moved to the right path

**Comment by rhtyd** [comment]
@davidjumani can you only keep changes strictly for #5338 and remove any other changes?

**Comment by davidjumani** [comment]
@rhtyd Fixing #5338 also partially fixes #3121 as well, so it is easier to get two issues fixed at once

**Comment by davidjumani** [comment]
@blueorangutan ui

**Comment by rhtyd** [comment]
@davidjumani check latest 4.15, upgrade path is now available.

**Comment by davidjumani** [comment]
@blueorangutan ui

**Comment by rhtyd** [comment]
Need one more review/test - @davidjumani @shwstppr @nvazquez @sureshanaparti...

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by shwstppr — APPROVED** [review]
Test upgrade from 4.15.1.
Config value updated to 20. Config is honored in the UI listviews

**Pull request merged**